astropy/visualization/mpl_normalize.py

```diff
@@ -202,13 +204,15 @@ def inverse(self, values, invalid=None):
         return values_norm * (self.vmax - self.vmin) + self.vmin
 
 
+@deprecated_renamed_argument("min_cut", "vmin", "6.1")
+@deprecated_renamed_argument("max_cut", "vmax", "6.1")
```

Comment:
This decorator doesn't need to be stacked 
```suggestion
@deprecated_renamed_argument(["min_cut", "max_cut"], ["vmin", "vmax"], ["6.1", "6.1"])
```

Reply:
Huh, didn't know you can use it like that.

Reply:
This is news to me too!  Thanks.